pytorch_lightning/trainer/trainer.py

```diff
@@ -296,6 +297,11 @@ def __init__(
                 In 'max_size_cycle' mode, the trainer ends one epoch when the largest dataset is traversed,
                 and smaller datasets reload when running out of their data. In 'min_size' mode, all the datasets
                 reload when reaching the minimum length of datasets.
+
+            stochastic_weight_avg: Whether to activate Stochastic Weight Averaging (SWA).
+                For an explanation on SWA, please take a look at:
+                `<https://pytorch.org/blog/pytorch-1.6-now-includes-stochastic-weight-averaging>`.
+
```

Comment:
Should there also be docs for it in trainer.rst file?

Reply:
You can also combine text and link like this:
``` `Stochastic Weight Averaging (SWA) <https://pytorch.org/blog/pytorch-1.6-now-includes-stochastic-weight-averaging/>_` ```

Reply:
Done !